azure/create-platform-prerequisites/README.md

```diff
@@ -1,72 +1,100 @@
-
-- Azure App Registration for the Cosmo Tech Platform
-  - Api permissions : `Platform.Admin` as Application on Cosmo Tech Platform API
-- Azure App Registration for AKS Network and Azure Digital Twins
-  - IAM roles : `Azure Digital Twins Data Owner` on Azure Digital Twins and `Network Contributor` on the Virtual Network
-- Azure App Registration for Cosmo Tech API Swagger UI
-  - API permissions : Delegated on Cosmo Tech Platform API
-- Azure App Registration for Restish (Optional)
-  - API permissions : Delegated on Cosmo Tech Platform API
-- Azure App Registration for WebApp (Optional)
-  - API permissions : Delegated on Cosmo Tech Platform API, `Workspace.Read.All` on Power BI and `Reports.Read.All` on Power BI
-- Azure Virtual Network for AKS
-- Azure public IP for the Cosmo Tech Platform
-- Azure DNS record
-- Role assignments for the Cosmo Tech Platform
-
-
+# Terraform script for Azure Prerequisites - Cosmo Tech Platform v2.3.5
+
+## Description
+
+This documentation describes the Azure prerequisite infrastructure needed to install the Cosmo Tech Simulation Platform using Terraform. The Terraform script creates Azure resources, app registrations and specific Cosmo Tech Simulation Platform roles. Here is the detail of the resources created
+
+* Azure Virtual Network for AKS (with one subnet, range /26)
+* Azure public IP for the Cosmo Tech Platform
+* Azure DNS record (type A on Public IP)
+
+* Azure App Registration for the Cosmo Tech Platform
+  * Authentication
+    * Supported account types: Single tenant
+    * Platform configurations: 
+      * Type: Web
+      * Redirect URIs: `platform-api-url`/v2/swagger-ui/oauth2-redirect.html (e.g. https://lab.api.cosmo-platform.com/v2/swagger-ui/oauth2-redirect.html)
+  * Secrets
+    * Create and save a secret
+  * Expose an API
+    * Add a scope
+      * Application ID URI: keep default value, should be `api://app-registration-client-id`
+      * Scope name: `platform`
+      * Who can consent: `Admins and users`
+      * Admin consent display name: `Cosmo Tech Platform Impersonate`
+      * Admin consent description: `Allow the appication to use the Cosmo Tech Platform with user account`
+  * Manifest
+    * In the manifest, replace appRoles property value with the [app-roles.json file](/home/jplanchenault/src/cosmotech-terraform/azure/create-platform-prerequisites/app-roles.json)
+  * API permissions
+    * API: `Microsoft Graph`
+      * Permission name: `User.Read`
+      * Type: `Delegated`
+      * Admin consent required
+    * API: `Azure App Registration for the Cosmo Tech Platform`
+      * Permission name: `Platform.Admin`
+      * Type: `Application`
+      * Admin consent required
+* Azure App Registration for AKS Network (previously used for ADT authentication)
+  * Secrets
+    * Create and save a secret
+  * RBAC
```

Comment:
RBAC ou IAM ?

Reply:
RBAC :)
I will change to role assignment.